src/corehost/build.cmd

```diff
@@ -137,8 +144,6 @@ goto :Failure
 :: Build the project created by Cmake
 set __msbuildArgs=/p:Platform=%__BuildArch% /p:PlatformToolset="%__PlatformToolset%"
 
-cd %__rootDir%
```

Comment:
why don't you need the CD anymore?

Reply:
I couldn't find any reason we needed the cd. We weren't using any relative paths in this script and we already `pushd` for calling `gen-buildsys-win.bat`, which is the only script that needs a specific working directory.